firefox/guide/firefox/firefox_preferences-install_extensions/oval/firefox.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="compliance" id="firefox_preferences-install_extensions"  version="1">
+    <metadata>
+      <title>Extensions install must be disabled.</title>
```

Comment:
Please change title to `Disable Extension Installation`

Reply:
Edited the title accordingly